src/pages/category/Category.jsx

```diff
@@ -1,9 +1,40 @@
-import { Component } from 'react';
+import React, { Component } from 'react';
+import ProductCard from './Components/ProductCard/ProductCard';
 import './Category.scss';
 
 class Category extends Component {
+  constructor() {
+    super();
+    this.state = {
+      product: [],
+    };
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/data/mockdata.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ product: data.data });
+      });
+  }
+
   render() {
-    return <p> 카테고리 페이지</p>;
+    return (
+      <div clsaaName="productContainer">
+        {this.state.product.map(product => {
+          return (
+            <ProductCard
+              key={product.id}
+              img={product.img}
+              series={product.series}
+              title={product.title}
+              mood={product.mood}
+              volume={product.volume}
+              price={product.price}
+            />
```

Comment:
이렇게 이름 그대로 넘길 때는 아래와 같이 간단하게 할 수 있습니다!
```suggestion
            <ProductCard
               key={product.id}
               {...product}
            />
```